jwql/tests/test_data/dark_monitor/make_test_data.py

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env python
+
+"""Create files for dark monitor testing"""
```

Comment:
@bhilbert4 We should be consistent and add `Authors` and `Use` sections in the docstring here.

Reply:
Oops. I didn't mean to actually include this in the repo. I used it to quickly generate some test data for a couple of the unit tests. Should I keep it in here, or remove it entirely?

Reply:
@bhilbert4 If you think we won't likely use it again, probably best to remove it so there's less code to maintain.

Reply:
Seems unlikely. I'll remove it.